openpype/hosts/photoshop/plugins/publish/extract_review.py

```diff
@@ -224,6 +224,7 @@ def _generate_thumbnail(
             "stagingDir": staging_dir,
             "tags": ["thumbnail", "delete"]
         })
+        instance.data["thumbnailPath"] = thumbnail_path  # for Ayon integration
```

Comment:
It is used for OpenPype too
```suggestion
        instance.data["thumbnailPath"] = thumbnail_path
```